.github/workflows/go_lint.yml

```diff
@@ -26,7 +26,8 @@ jobs:
         uses: golangci/golangci-lint-action@v5
         with:
           version: v1.59.0
-          args: --config .golangci.toml --max-same-issues=0 --max-issues-per-linter=0 --verbose
+          args: |
+            --config .golangci.toml --max-same-issues=0 --max-issues-per-linter=0 --verbose ./pkg/... ./pkg/apimachinery/... ./pkg/apiserver/... ./pkg/build/wire/... ./pkg/promlib/... ./pkg/util/xorm/...
```

Comment:
🤔 It looks like the additional items after  `./pkg/...`  are redundant - am I seeing things?

Reply:
no, these are all independent modules, and because `golangci-lint` follows the same rules as `go mod` we have to be explicit (this is what's also listed in the `golangci-lint` make target, effectively)